DigitalLearningSolutions.Web/Views/TrackingSystem/Delegates/ActivityDelegates/RemoveDelegateSelfAssessment.cshtml

```diff
@@ -0,0 +1,58 @@
+﻿@using DigitalLearningSolutions.Web.ViewModels.TrackingSystem.Delegates
+@using Microsoft.Extensions.Configuration
+@model DelegateSelfAssessmenteViewModel;
+@inject IConfiguration Configuration;
+@{
+    var errorHasOccurred = !ViewData.ModelState.IsValid;
+    ViewData["Title"] = (errorHasOccurred ? "Error: " : "") + "Delegate Self Assessment - Confirm Remove";
+}
+
+@section NavMenuItems {
+    <partial name="Shared/_NavMenuItems" />
+}
+
+<div class="nhsuk-grid-row">
+    <div class="nhsuk-grid-column-full word-break">
+        @if (errorHasOccurred)
+        {
+            <vc:error-summary order-of-property-names="@(new[] { nameof(Model.ActionConfirmed) })" />
+        }
+        <h1 id="page-heading">Confirm Remove Delegate Self Assessment</h1>
+        <div class="nhsuk-grid-row">
+            <div class="nhsuk-grid-column-one-quarter nhsuk-heading-l">
+                <div class="nhsuk-u-font-weight-bold">
+                    Delegate self assessment:
+                </div>
```

Comment:
Have a separate row for "Self Assessment:" and load the self assessment name over there. See the screen for remove from course for this.